pynsee/macrodata/_download_idbank_list.py

```diff
@@ -1,18 +1,17 @@
 # -*- coding: utf-8 -*-
 # Copyright : INSEE, 2021
 
-import pandas as pd
-import os
 
 from pynsee.macrodata._dwn_idbank_files import _dwn_idbank_files
 
 from pynsee.utils._get_credentials import _get_credentials
 from pynsee.utils.save_df import save_df
 
+
 @save_df(day_lapse_max=90)
 def _download_idbank_list(update=False, silent=True, insee_date_test=None):
 
-    keys = _get_credentials()
+    _get_credentials()
```

Comment:
I think this line could be removed

Reply:
I was not sure (because of the environemnt variables), but it should in any case when we merge the PR with the new session object.